src/u2f/u2f_app.c

```diff
@@ -7,15 +7,37 @@
 
 #include <stddef.h>
 #include <stdio.h>
+#include <wally_bip39.h>
 
 #define APPID_BOGUS_CHROMIUM "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
 #define APPID_BOGUS_FIREFOX "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
 
+#define SHORT_MNEMONIC_LENGTH 4
```

Comment:
I propose a more descriptive name: `SHORT_MNEMONIC_NUM_WORDS`.

the current name is ambiguous, as it could refer to bytes, words, etc.

Reply:
done.